### PR TITLE
fix require error

### PR DESCRIPTION
**Pull request by huhu2356**
## What

- Modify the `main` attribute within `package.json` for all of `packages/*` to include the npm org name
- Bump patch version in preparation for release

## Why

fix error:
- use @rsksmart/rsk3 will throw error:
```
internal/modules/cjs/loader.js:605
    throw err;
    ^

Error: Cannot find module '@rsksmart/rsk3'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:603:15)
    at Function.Module._load (internal/modules/cjs/loader.js:529:25)
    at Module.require (internal/modules/cjs/loader.js:657:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/Users/crazyguy/work/my-test/test.js:1:76)
    at Module._compile (internal/modules/cjs/loader.js:721:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:732:10)
    at Module.load (internal/modules/cjs/loader.js:620:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:560:12)
    at Function.Module._load (internal/modules/cjs/loader.js:552:3)
```

this PR is my try, need to verify and test it;

## Refs

- [Task](https://trello.com/c/ruv3rg8A/301)

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -92,14 +92,6 @@ export default (name, outputFileName, globals, ignoreEsm) => {
...
             ]
         };
-
-        if (name === 'Web3') {
-            config[1].input = 'src/Web3.js';
-        }
-    }
-
-    if (name === 'Web3') {
-        config[0].input = 'src/Web3.js';
```
Comment (chrisli30) [inline rollup.config.js:102]: Do we need this line for src/rsk3.js?

**Comment by bguiz** [comment]
More details, with a variant of the error in the description:

```
Error: Cannot find module '${PROJECT_ROOT}/node_modules/@rsksmart/rsk3/dist/rsk3.cjs.js'. Please verify that the package.json has a valid "main" entry
```

```
$ ls node_modules/@rsksmart/rsk3/dist/@rsksmart/
rsk3.cjs.js  rsk3.esm.js  rsk3.umd.js
```

File: rollup.config.js (hunk 1 above)
Comment (bguiz) [reply rollup.config.js:102]: I have not verified the impact of this change, I assume that this is OK

**Comment by bguiz** [comment]
@huhu2356 this fixed the issue

- this has been released in [v0.3.3](https://www.npmjs.com/package/@rsksmart/rsk3/v/0.3.3)
- and I have tested that it is able to be require'd in the context of the [originally failure scenario](https://github.com/rsksmart/json-rpc-web3js/pull/3)

**Pull request merged**